<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="jaykishore-PC" failures="1" tests="12" name="Surefire test" time="8.853" errors="0" timestamp="2024-01-30T22:56:12 GMT-05:00">
  <testcase classname="Api.test.DDTests" name="testPostUser" time="2.981"/>
  <testcase classname="Api.test.DDTests" name="testPostUser" time="0.287"/>
  <testcase classname="Api.test.DDTests" name="testPostUser" time="0.307"/>
  <testcase classname="Api.test.DDTests" name="testPostUser" time="0.199"/>
  <testcase classname="Api.test.DDTests" name="testDeleteUserByName" time="0.202"/>
  <testcase classname="Api.test.DDTests" name="testDeleteUserByName" time="0.169"/>
  <testcase classname="Api.test.DDTests" name="testDeleteUserByName" time="0.179"/>
  <testcase classname="Api.test.DDTests" name="testDeleteUserByName" time="0.205">
    <failure type="java.lang.AssertionError" message="expected [200] but found [405]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at Api.test.DDTests.testDeleteUserByName(DDTests.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
  <testcase classname="Api.test.UserTests" name="testPostUser" time="0.231"/>
  <testcase classname="Api.test.UserTests" name="testGetUserByName" time="0.186"/>
  <testcase classname="Api.test.UserTests" name="testUpdateByName" time="0.372"/>
  <testcase classname="Api.test.UserTests" name="testDeleteUserByName" time="0.181"/>
</testsuite> <!-- Surefire test -->
